docs/datamodel/computables.rst

```diff
@@ -16,7 +16,9 @@ non-computable counterparts.
 Computables are useful in the situations where there is a frequent need for
 some value that is derived from the values of existing properties and links.
 
-For example:
+For example, here we define the ``User`` type to contain the
+``fullname`` computable property that is derived from user's first and
+last name:
```

Comment:
While you're here, please fix the example below: it should use `++` instead of `+`

Reply:
OK